src/frontend/.dockerignore

```diff
@@ -0,0 +1,11 @@
+# Items that don't need to be in a Docker image.
+# Anything not used by the build system should go here.
+Dockerfile
+.dockerignore
+.gitignore
+README.md
+
+# Artifacts that will be built during image creation.
+# This should contain all files created during `npm run build`.
+build
+node_modules
```

Comment:
CRLF. Please leave an empty line at the end of files to avoid triggering @akroon3r